bitnami/harbor/README.md

```diff
@@ -0,0 +1,348 @@
+# Helm Chart for Harbor
+
+This Helm chart has been developed based on [goharbor/harbor-helm](https://github.com/goharbor/harbor-helm) chart but including some features common to the Bitnami chart library.
+For example, the following changes have been introduced:
+
+- Possibility to pull all the required images from a private registry through the  Global Docker image parameters.
+- Redis and PostgreSQL are managed as chart dependencies.
+- Liveness and Readiness probes for all deployments are exposed to the values.yaml.
+- Uses new Helm chart labels formating.
+- Uses Bitnami container images:
+  - non-root by default
+  - published for debian-9, ol-7, and eventually centos-7
+- At this moment, this chart does not support the Harbor optional component Chartmuseum but it does support Clair and Notary integrations.
+
+## Introduction
+
+This [Helm](https://github.com/kubernetes/helm) chart installs [Harbor](https://github.com/goharbor/harbor) in a Kubernetes cluster. Welcome to [contribute](CONTRIBUTING.md) to Helm Chart for Harbor.
+
+## Prerequisites
+
+- Kubernetes cluster 1.10+
+- Helm 2.8.0+
+
+## Installation
+
+### Download the chart
+
+Download Harbor helm chart
+
+```bash
+git clone https://github.com/bitnami/charts
+```
+
+Change directory to Harbor code
+
+```bash
+cd charts/bitnami/harbor
+```
+
+### Configure the chart
```

Comment:
This looks awkward to me.. We have a section called `## Configuration`. Why don't we explain this stuff on that section?

Reply:
I have re-organized the whole README